Withings-SDK-iOS/WithingsMeasureAPIClient.m

```diff
@@ -234,8 +261,18 @@ + (MeasuresDateRange*)dateRangeBetweenStartDate:(NSDate*)startDate andEndDate:(N
 {
     NSDate *startDate = _startDate ? _startDate : [NSDate dateWithTimeIntervalSince1970:0];
     NSDate *endDate = _endDate ? _endDate : [NSDate date];
-    return @{@"startdate" : @([startDate timeIntervalSince1970]),
```

Comment:
Ok. The problem is that the parameters method of MeasuresDateRange is also used for body measures. So, could you create a specific method to get the datetime related to noon for sleep API only ?